main.py

```diff
@@ -897,3 +897,22 @@ def handle_link_maintenance_end(self, event):
             link.endpoint_a.enable()
             link.endpoint_b.enable()
             self.notify_link_status_change(link)
+
+    @listen_to('kytos/maintenance.start_uni')
+    def handle_uni_maintenance_start(self, event):
+        """Deal with the start of UNIs maintenance."""
+        maintenance_unis = event.content['unis']
+        for maintenance_uni in maintenance_unis:
+            maintenance_uni.interface.disable()
+            if maintenance_uni.interface.is_active():
+                self.handle_link_down(maintenance_uni.interface)
+        self.notify_topology_update()
+
+    @listen_to('kytos/maintenance.end_uni')
+    def handle_uni_maintenance_end(self, event):
+        """Deal with the end of links maintenance."""
+        maintenance_unis = event.content['unis']
+        for maintenance_uni in maintenance_unis:
+            maintenance_uni.interface.enable()
```

Comment:
I think that the interface state should be saved. If it was in the disabled state before the maintenance it must remain disabled.